<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesEditor</class>
 <widget class="QWidget" name="PropertiesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Material</string>
     </property>
     <property name="buddy">
      <cstring>materialSelector</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="materialSelector">
     <item>
      <property name="text">
       <string>lead</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>steel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ice</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wood</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rubber</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>foam</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="static_material">
     <property name="text">
      <string>Static</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
